src/UI/Seller/src/web.config

```diff
@@ -20,7 +20,7 @@
         <add name="Cache-Control" value="no-cache, no-store"/>
         <add name="X-Content-Type-Options" value="nosniff" />
         <add name="Referrer-Policy" value="no-referrer-when-downgrade"/>
-        <add name="Content-Security-Policy" value="default-src 'self'; connect-src *; font-src *; frame-src *; img-src *; data: https; media-src *; object-src *; script-src * 'unsafe-inline' 'unsafe-eval'; style-src * 'unsafe-inline';" />
+        <add name="Content-Security-Policy" value="default-src 'self'; connect-src *; font-src *; frame-src *; img-src * 'self'; data: https; media-src *; object-src *; script-src * 'unsafe-inline' 'unsafe-eval'; style-src * 'unsafe-inline';" />
```

Comment:
Wasn't this removed to fix Cors issues or something else with assets? or is that fixed now with this change?

Reply:
Yeah honestly I can't remember I just went and tried a bunch of uploads and it works with that config. I do think at one point we were having cors issues but that doesn't seem to be the case anymore (possibly was due to the fact that we were using the headstart sdk for assets but now that is replaced with the correct cms url)